CPS-0001/README.md

```diff
@@ -0,0 +1,47 @@
+
+## **Abstract**
+This document attempts to describe the problem in the Cardano ecocystem where there are many different types **subjects** or *chain entities* and there's a need to associate metadata with them. This metadata can be used to describe an script, stake pools, script hashes, token policies and applications. This metadata can be used to provide information to the user on it's applications, it's trustworthiness and how to interact with it.
+
+## **Goals**
+- Define how metadata can be associated with the a subject (i.e dapp, stake pool, token policy, etc)
+- Metadata should be discoverable by wallets and applications
+- Associate some form of identity to a metadata claim
+
+## **Use Cases**
+- Find scripts used by a dApp
+- Discover general information of a dApp (i.e name, description, icons, etc)
+- Find the different metadata claims associated with a subject
+- Associate some form of identity to a claim's owner so that it can be verified cryptographically
+- Offer mechanism to attest for the correctness of a given metadata object that can be fetched by wallets and applications from off-chain sources (i.e CIP-26 complaint servers)
+- Discover datum schemas used by a script in a specific context or dApp use cases
+- An wallet receiving a request to connect to a dApp, it can verify the authenticity of the dApp and the metadata associated with it.
+
+## **Problem**
+**`Discoverability`**: Means to discover the different metadata claims associated with a subject. Discoverability is important for wallets, applications (i.e dapps, stores, etc) and users to be able to find the different metadata claims associated with a subject. This is important for the user to be able to make an informed decision on how to interact with a subject.
+
+**`Correctness`**: Lack of mechanism to assert that a given metadata claim is correct. Anybody and any service can provide metadata structures, but it's necessary to attest for the correctness of this metadata so that the user (or applications) are be able to make an informed decision in accepting, rejecting or how to interact with a subject.
+
+**`Trust`**: There's no standard mechanism to assert that the metadata is coming from a legitimate source. This is important for the user to be able to make an informed decision on how to interact with a subject. This is also important for applications to be able to provide a better user experience. 
+
+## **Open Questions**
+- Is this a combination of problems and should be split into multiple CPS?
+- Does this englobes all the problems related to metadata?
+- Trust can be anchored to the owner's metadata or also third-parties that attest for the correctness of the metadata. How to handle this?
+- How to associate identity to a metadata claim?
+- How to handle the case where a subject has multiple metadata claims associated with it?
+- How to handle the case where a subject has multiple metadata claims associated with it and the user wants to select a specific one?
```

Comment:
Please re-order the sections as per the latest structure 🙏  (and apologies for this, I guess that goes with being one of the first CPS pioneers!):

- Abstract
- Problem
- Use Cases
- Goals
- Open Questions

--- 

Also, given the numerous proposed solutions; it might help to formulate what part of the problem remains to be solved and what parts have been properly addressed already.

Reply:
> Also, given the numerous proposed solutions; it might help to formulate what part of the problem remains to be solved and what parts have been properly addressed already.

Great @KtorZ ; i think here there some attempts in coming out with some solutions (i.e token metadata, off chain storage[cip-26]), for very specific use cases but not to the general problem of discovering, verifying and asserting trust in metadata? 

Maybe someone can help me by pointing out current solutions / implementations, that in their opinion, are working well for a specific context. And those can provide lessons to try tackle this in a more generalized way.

CC: @rphair @SebastienGllmt

Reply:
What I meant was that, the current document lists several CIPs as "proposed solutions"; which implies that the listed proposals address either partially or fully the problem. If they are simply _related_ to the problem, but not actual solutions to it, then I'd suggest to move them elsewhere (i.e.  in the problem statement itself, if those existing efforts and their shortcomings are worth mentioning). 

(note: I haven't reviewed the statement yet, nor any of the linked PR; so I am mostly speculating here. My gut feeling being that this particular CPS is about some dimensions of metadata -- discoverability and trust -- that existing CIPs do not address properly.